dashboard/src/features/shared/routes.js

```diff
@@ -2,7 +2,7 @@ import { RoutingContainer } from 'features/shared/components'
 import { humanize } from 'utility/string'
 import actions from 'actions'
 
-const makeRoutes = (store, type, List, New, Show, options = {}) => {
+const makeRoutes = (store, type, List, New, Show, Update, options = {}) => {
```

Comment:
I suspect this will break any invocation of `makeRoutes` that passes a value for `options`. Can you double-check to make sure?